packages/moonstone/Panels/Viewport.js

```diff
@@ -128,17 +128,21 @@ const ViewportBase = class extends React.Component {
 	)
 
 	mapChildren = (children, generateId) => React.Children.map(children, (child, index) => {
-		const {spotlightId = generateId(index, 'panel-container', Spotlight.remove)} = child.props;
-		const props = {
-			spotlightId,
-			'data-index': index
-		};
-
-		if (child.props.autoFocus == null && this.state.direction === 'forward') {
-			props.autoFocus = 'default-element';
+		if (child) {
+			const {spotlightId = generateId(index, 'panel-container', Spotlight.remove)} = child.props;
+			const props = {
+				spotlightId,
+				'data-index': index
+			};
+
+			if (child.props.autoFocus == null && this.state.direction === 'forward') {
+				props.autoFocus = 'default-element';
+			}
+
+			return React.cloneElement(child, props);
+		} else {
+			return null;
```

Comment:
I wonder if other `falsy` values other than `null` would give different results.  Empty string?  The number 0?  This seems like a potential (albeit very, very small) oversight.  Maybe we should return `child`?  Leaving this here as food for thought.